File: 백준/Silver/11651. 좌표 정렬하기 2/좌표 정렬하기 2.js
const input = require('fs').readFileSync(0, 'utf-8').toString().trim().split('\n').map(t => t.split(' ').map(Number));
const [N] = input[0];
const nums_sorted = input.slice(1).sort((a, b) => a[1] === b[1] ? a[0] - b[0] : a[1] - b[1]);
console.log(nums_sorted.map(t => t.join(' ')).join('\n'));

File: 백준/Silver/11651. 좌표 정렬하기 2/좌표 정렬하기 2.py
N = int(input())
coords = []

for _ in range(N):
    x, y = map(int, input().split())
    coords.append([x, y])

coords.sort(key=lambda x: (x[1], x[0]))

for i in range(N):
    print(f'{coords[i][0]} {coords[i][1]}')